src/modules/form/components/DynamicField.tsx

```diff
@@ -294,7 +301,7 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
             <MinutesDurationInput
               value={isNil(value) ? '' : value}
               onChange={onChangeValue}
-              inputWidth={width}
```

Comment:
`inputWidth` was not being used here, since MinutesDurationInput sets the width of the input fields directly